README.md

```diff
@@ -4,3 +4,12 @@ This component was extracted out of SonarQube, and is released independently sin
 
 ## Developing plugins
 See documentation [here](https://docs.sonarqube.org/latest/extend/developing-plugin/) about how to use the `sonar-plugin-api` to develop plugins.
+
+## Dependency
+The API was relocated when it was extracted out of SonarQube. Its new coordinates are:
+```
+org.sonarsource.api.plugin:sonar-plugin-api:9.8.0.203
```

Comment:
What about having the "new coordinates" without a version? It will be hard to keep it outdated. Instead of that we could have:
```
 org.sonarsource.api.plugin:sonar-plugin-api:<version>
 ```

Reply:
good idea